apps/api-gateway/scripts/generate_ops_analysis_report.py

```
@@ -38,21 +38,77 @@
+async def main(start_date: date, end_date: date, brand_ids: list, output_dir: str,
+               split_by_brand: bool = False):
     """生成报告主流程"""
     from src.core.database import get_db_session
-    from src.services.ops_analysis_report_service import OpsAnalysisReportService
+    from src.services.ops_analysis_report_service import (
+        OpsAnalysisReportService,
+        SEED_MERCHANTS,
+    )
 
     print(f"\n{'='*60}")
     print(f"  屯象OS · 多品牌运营分析报告生成器")
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)